app/Enums/ChangeInFavouriteCityEnum.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+enum ChangeInFavouriteCityEnum: string
+{
+    case Added = "added to";
+    case Removed = "removed from";
```

Comment:
```suggestion
    case Added = "added_to";
    case Removed = "removed_from";
```

Reply:
Wouldn't it display also in notification?

Reply:
yep but maybe use translation for things like that